tapdance/assets.go

```diff
@@ -163,6 +190,15 @@ func (a *assets) readConfigs() {
 	} else {
 		Logger.Infoln("Pubkey succesfully read from " + pubkeyFilename)
 	}
+
+	// TODO: stop printing backoff read failures
```

Comment:
Right, this is purely temporary.
It wouldn't even make it to PR, if it wasn't so late